fog/api/proto/view.proto

```diff
@@ -267,4 +255,10 @@ message TxOutRecord {
     ///
     /// Represented as seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z.
     fixed64 timestamp = 7;
+    /// The crc32 of the commitment data bytes.
+    /// This is a 4-byte IEEE crc32 of the bytes of the tx_out_amount_commitment_data bytes, which is present if
+    /// the full tx_out_amount_commitment_data is omitted.
+    /// The client can recompute the tx_out_amount_commitment from the other data that we include.
+    /// They can confirm correct recomputation by checking this crc value.
+    fixed32 tx_out_amount_commitment_data_crc32 = 8;
```

Comment:
this change to the proto is the main point of the change -- the fog ingest enclave will only set this crc32 field and leave the earlier commtiment_data field empty.

Reply:
the crc32 is also possibly not necessary, but kyle suggested that we should retain this in order to keep the same level of "checkability" as before, and this only adds a few bytes